ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -1935,3 +1935,35 @@ spec:
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS6.3.6
   contributors: artemist-work
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Secure Keyboard Entry Terminal.app Is Enabled
+  platforms: macOS
+  platform: darwin
+  description: |
+    Secure Keyboard Entry prevents other applications on the system and/or network from detecting and recording what is typed into Terminal. Unauthorized applications and malicious code could intercept keystrokes entered in the Terminal.
+    Enabling Secure Keyboard Entry minimizes the risk of a key logger from detecting what is entered in Terminal.
+  resolution: |
+    Profile Method:
+    Create or edit a configuration profile with the following information:
+      1. The PayloadType string is com.apple.Terminal 
+      2. The key to include is SecureKeyboardEntry
+      3. The key must be set to
+      <true/>
+  query: |
+    SELECT 1 WHERE  EXISTS (
+        SELECT * FROM plist      
+        WHERE path LIKE '/Library/Managed Preferences/%/com.apple.Terminal.plist'     
+        AND key = 'SecureKeyboardEntry' 
+        AND value == 1
+    ) AND NOT EXISTS (
+        SELECT * FROM plist      
+        WHERE path LIKE '/Library/Managed Preferences/%/com.apple.Terminal.plist'     
+        AND key = 'SecureKeyboardEntry' 
+        AND value != 1
+    );
```

Comment:
If you are using a test profile which they recommend:
```
Note: Since the profile method sets a system-wide setting and not a user-level one, the profile method is the preferred method. It is always better to set system-wide than per user.
```
then this query should use `managed_policies` instead.